src/main/java/ru/mail/polis/homework/objects/CustomArrayWrapper.java

```diff
@@ -23,6 +28,7 @@ public CustomArrayWrapper(int size) {
     public void add(int value) {
         checkIndex(position);
         array[position] = value;
+        modCount++;
```

Comment:
Но ведь modCount должен увеличиваться только при структурных изменениях массива, к которым изменение элемента по индексу не относится, разве нет?

Javadoc к ArrayList:
"A structural modification is any operation that adds or deletes one or more elements, or explicitly resizes the backing array; merely setting the value of an element is not a structural modification."

Javadoc к modCount:
"The number of times this list has been structurally modified. Structural modifications are those that change the size of the list, or otherwise perturb it in such a fashion that iterations in progress may yield incorrect results."